lib/CMakeLists.txt

```diff
@@ -9,3 +9,4 @@ add_subdirectory(os)
 add_subdirectory(util)
 add_subdirectory_ifdef(CONFIG_SMF smf)
 add_subdirectory_ifdef(CONFIG_OPENAMP              open-amp)
+add_subdirectory(bluetooth)
```

Comment:
Does the library have to be in the `lib` folder or can it be moved to `subsys/bluetooth/host` so that it is in the same place as the rest of the bluetooth stack?

Reply:
`subsys/bluetooth/host` should be fine, I think. When this was originally described as a library (before I saw the pull request) I assumed that just meant that it would be a layer that builds on top of existing public Bluetooth APIs rather than using internal APIs, but I didn't expect to see something added to `lib/bluetooth/`

Reply:
I'd like to keep it separated. Maybe `subsys/bluetooth/lib/` would be appropriate, we just thought that the root `lib/` directory existed for that exact purpose.

Reply:
Not sure if there is any official guideline here, but the existing libraries in `lib/` seem to be general purpose. I would like to have all bluetooth related code in `subsys/bluetooth`.

Reply:
Yes, this will go under `subset/bluetooth/`. The main open question is whether it should be under `host` or some other directory there. I suppose adding a `lib` directory as suggested by @jori-nordic might make sense, but we should also look at what we already have and see if it makes sense to group this new functionality with existing Bluetooth sub-modules that can also be considered libraries (in the sense that there'd be multiple "library submodules" under the same directory.